lib/dry/cli.rb

```diff
@@ -36,16 +36,19 @@ def self.command?(command)
 
     # Create a new instance
     #
-    # @param registry [Dry::CLI::Registry, Block] a registry or a configuration block
+    # @param command_or_registry [Dry::CLI::Registry, Dry::CLI::Command] a registry or singular command
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Line is too long. [103/100]](https://app.codacy.com/gh/dry-rb/dry-cli/pullRequest?prid=4800973)